test/toolchain/clang-libc++.cmake

```diff
@@ -13,7 +13,11 @@ set(INT128_ENABLED_FLAGS "-DCNL_USE_INT128=1")
 set(INT128_DISABLED_FLAGS "-DCNL_USE_INT128=0")
 
 set(SANITIZE_ENABLED_CXX_FLAGS "-fsanitize=address,undefined -fsanitize-trap=undefined -g -O0")
-set(SANITIZE_ENABLED_LINKER_FLAGS "-fsanitize=address,undefined -rtlib=compiler-rt -lgcc_s")
```

Comment:
The `-lgcc_s` thing is entirely me flailing around and pasting random code from StackOverflow until a problem went away. I'm confident I could find a way to avoid this.

Reply:
I noticed that you can actually drop that -lgcc_s library completely from clang-libc++.cmake and compile with clang-11 with -DCNL_SANITIZE=ON. I do not know, however, whether that combo makes sense. This is hugely fast compilation (10x at least) compared to similar complilation on gcc which makes me suspicious.

Reply:
Bear in mind I recently enabled ccache in the pipeline. Expect builds to sometimes take 3m instead of 10m+.